src/NuGet.Core/NuGet.Packaging.Core/NuGet.Packaging.Core.csproj

```diff
@@ -0,0 +1,27 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <Description>The (former home to) core data structures for NuGet.Packaging.</Description>
+    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFramework />
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <PackProject>true</PackProject>
+    <Shipping>true</Shipping>
+    <IncludeInVSIX>true</IncludeInVSIX>
```

Comment:
This should be removed. 
It ends up including the localization assets from the nuget.packaging.core.dll in VSIX. 

The nuget.packaging.core.dll is not there in the VSIX (and as far as I know it doens't need to be)